packages/filebrowser/src/browser.ts

```diff
@@ -252,6 +253,10 @@ export class FileBrowser extends SidePanel {
   private async _createNew(
     options: Contents.ICreateOptions
   ): Promise<Contents.IModel> {
+    // normalize the path if the file is created from a custom drive
```

Comment:
Is there an easy way to add a unit test for this?

Reply:
Hoping so yes.

Gonna first rebase to grab the UI tests fixes from https://github.com/jupyterlab/jupyterlab/pull/15524, to make sure this change does not break other things.

Reply:
Just following up on the question about a unit test here - do you think it could be added in this PR?

Reply:
So the UI tests are now passing after https://github.com/jupyterlab/jupyterlab/pull/15524.

Hopefully it should be possible to add a simple unit test yes. Maybe would need to double check if the current testutils mock correctly handle drives too.

Reply:
Did you plan to add a unit test in this PR or would you prefer it to be merged without a test?